chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -1406,6 +1407,18 @@ impl Get<u64> for TimestampProvider {
     }
 }
 
+parameter_types! {
+    // Chain ID of Holesky.
+    pub RewardsEthereumSovereignAccount: AccountId =
```

Comment:
Design wise, We should have one sovereign account for both rewards and token transfer to ethereum.

From what I understand, having one sovereign account for ethereum make more sense as having different sovereign accounts for rewards and token transfer will not work once we implement token transfer from ethereum to starlight.

Reply:
For now we don't know so let's keep it like that. If we take the decision to have the same account we'll change the name and use it in multiple places.

Reply:
What I mean is it will need to be one account if we are ever going to have token transfer. 

Otherwise there can be accounting mismatch.

Reply:
I agree that the naming can change to something more generic like `EthereumSovereignAccount`. not critical since we can change it later